README.md

```diff
@@ -8,6 +8,7 @@ Blogging my baking expeditions. #bakingexpeditions #noms
     cd baking-expeditions
     npm install # Install dependencies from package.json
     webpack # Generate JS bundle
+    gulp sass # Build CSS
     npm start
 
```

Comment:
I'm trying to understand the goals of gulp (task-runner) and webpack (package-bundler). I read the post you linked and also linked a stackoverflow question below. It seems like something like SASS compilation and CSS bundling might be well-suited for webpack? While running the bundler (webpack), lint, tests might be good for gulp.

I think the gulp stuff looks cool! It just seems confusing to me to have too many things going on :/
No need to change it though, I know you're just playing around with some new technologies :) I might try and do the SASS stuff on webpack. And then when we get to tests, maybe I'll use gulp to run both webpack and tests.

[task runners vs package bundlers](http://stackoverflow.com/questions/33561272/task-runners-gulp-grunt-etc-and-bundlers-webpack-browserify-why-use-toge)